/nmea_publisher:
  ros__parameters:
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    nmea_ip: 192.168.0.50
    nmea_port: 9095
    use_sim_time: false
